CHANGELOG.rst

```diff
@@ -1,5 +1,18 @@
 
 ----------
+2015-06-09
+----------
```

Comment:
I don't know whats the proper style for changelogs, but @evildmp proposed: https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/CHANGELOG.rst